src/components/BrandedTooltip.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <s-tooltip
+    v-bind="{
+      animation: 'none',
+      popperClass: 'wallet-tooltip',
+      ...$attrs
+    }"
+  >
+    <slot slot="content" name="content"/>
+    <!-- element-ui disabled tooltip disables wrapped button, this is the way to fix it -->
+    <span>
+      <slot/>
+    </span>
+  </s-tooltip>
+</template>
+
+<script lang="ts">
+export default {}
+</script>
+
+<style lang="scss">
+.el-tooltip__popper.is-light.wallet-tooltip {
+  padding: 4px 8px 6px;
```

Comment:
As I mentioned in another place, it's better to use variables instead of static values.